scala-package/macros/src/main/scala/org/apache/mxnet/GeneratorBase.scala

```diff
@@ -206,16 +239,24 @@ private[mxnet] trait RandomHelpers {
     )
   }
 
-  // hacks to manage the fact that random_normal and sample_normal have
-  // non-consistent parameter naming in the back-end
-  // this first one, merge loc/scale and mu/sigma
+  /**
+    * hacks to manage the fact that random_normal and sample_normal have
```

Comment:
Could `random_normal` and `sample_normal` get some kind of cross-link or should it have ticks like this?

Reply:
These are internal method names used on C backend. The method used in here is essentially combine them in the front end